Automattic/wp-calypso
# Hosting Command Palette: Embed the command palette on client/layout

- Closes https://github.com/Automattic/dotcom-forge/issues/4572

## Proposed Changes

* In this PR we embed the WPcom `Command Palette` in `client/layout` so it will be available across all calypso pages that use that layout. It is behind a feature flag, but we will remove it in the future.

## Testing Instructions



* Access http://calypso.localhost:3000/log-in
* Press `cmd+k` on Mac or `ctrl+k` on Windows
* Observe the command palette is not available
* Access `http://calypso.localhost:3000/post/$siteSlug`
* Observe the command palette displayed is the Gutenberg version
* Access `http://calypso.localhost:3000/sites`
* Observe the command palette opens correctly with the shortcut
* Navigate to other page using the command palette, like `Open my profile` or `Open hosting configuration`
* Observe the command palette opens correctly with the shortcut
* Try in Jetpack cloud and confirm the command palette is not available

## Pre-merge Checklist
